scripts/site/_site/doc/app/footer/footer.component.ts

```diff
@@ -89,20 +89,23 @@ import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnInit, Output
     </footer>
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
-  styleUrls: ['./footer.component.less']
+  styleUrls: ['./footer.component.less'],
+  host: {
+    ngSkipHydration: ''
+  }
```

Comment:
The footer component fails to hydrate, seems to be an angular bug, use the ngSkipHydration flag to make the footer component skip the hydration.